test/user.test.js

```diff
@@ -401,4 +401,169 @@ describe('USER TEST SUITE', () => {
         expect(response.body.error).to.equal('Invalid token');
       });
   });
+
+  describe('User Update Validations', () => {
+    it('should return No token provided, when No token is provided',
```

Comment:
Please change the wording of this spec. It is better written as:

`Should not allow update when no token is provided`

You don't need to write the exact error message in the spec description.

Reply:
ok